src/renderers/dom/shared/ReactBrowserEventEmitter.js

```diff
@@ -151,6 +152,41 @@ var topEventMapping = {
   topWheel: 'wheel',
 };
 
+var localOnly = {
+  topAbort: true,
+  topCanPlay: true,
+  topCanPlayThrough: true,
+  topDurationChange: true,
+  topEmptied: true,
+  topEncrypted: true,
+  topEnded: true,
+  topError: true,
+  topInvalid: true,
+  topLoad: true,
+  topLoadedData: true,
+  topLoadedMetadata: true,
+  topLoadStart: true,
+  topPause: true,
+  topPlay: true,
+  topPlaying: true,
+  topProgress: true,
+  topRateChange: true,
+  topReset: true,
+  topSeeked: true,
+  topSeeking: true,
+  topStalled: true,
+  topSubmit: true,
+  topSuspend: true,
+  topTimeUpdate: true,
+  topToggle: true,
+  topVolumeChange: true,
+  topWaiting: true,
+  topTouchMove: true,
+  topTouchStart: true,
+  topTouchCancel: true,
+  topTouchEnd: true,
```

Comment:
@spicyj I've added touch cancel and touch end to the list here ☝️. They'll get attached locally now.